Loops/loops_homework_6.py

```diff
@@ -0,0 +1,49 @@
+list_some_elements = [23, 0, 56, 100, 50, True, False, 'abc', 10.5]
+client_num = int(input("Please, enter your number: "))
+dividing_list_num = []
+for elements in list_some_elements:
+    if type(elements) == int:
+        dividing_num = elements // client_num
+        dividing_list_num.append(dividing_num)
+
+print(f"After dividing your number {client_num} by numbers from list {list_some_elements} "
+      f"you get numbers in list {dividing_list_num}")
+
+
+unsorted_list = [1, 5, 68, 0]
+for num in range(1, len(unsorted_list)):
+    if unsorted_list[num] != unsorted_list[num - 1] + 1:
+        print(f" First inconsistent number in list {unsorted_list} is {unsorted_list[num]}")
+        break
+
```

Comment:
You forgot about else block(
![image](https://github.com/Pticatos86/test_repos_python/assets/51053483/019c5976-7ab3-43c1-9e57-4650ed3eb4cc)

Reply:
add else block